lib/blacklight/path.rb

```diff
@@ -0,0 +1,129 @@
+module Blacklight
+  class Path
```

Comment:
Add class documentation: This class decorates ActiveController::Params to provide methods to operate upon the parameters that serialize the blacklight state, namely: `:f, :q, :page, :per_page and, :sort`